test/spoke/integration/Spoke.t.sol

```diff
@@ -1060,6 +1020,7 @@ contract SpokeUpdateContract is BaseTest, SpokeTestHelper {
         spoke.update(PoolId.wrap(0), ShareClassId.wrap(0), bytes(""));
     }
 
+    /*
```

Comment:
I commented these tests, because I want to fix them in an incoming PR where I want to move the UpdateContract processing to `MessageProcessor`, which fits better. Will upload this soon